### PR TITLE
Unity 2020.3.25 w/ CloudRendering Engine

### PR DESCRIPTION
**Pull request by ekolve**
This PR upgrades the version of Unity to 2020.3.25 and also adds the CloudRendering (headless rendering).

### CODE REVIEW

**Review by winthos — APPROVED** [review]
lgtm

File: tasks.py [hunk 1]
```diff
@@ -1120,6 +1117,28 @@ def ci_build(context):
     lock_f.close()
 
 
+@task
+def install_cloudrendering_engine(context, force=False):
+    if not sys.platform.startswith("darwin"):
+        raise Exception("CloudRendering Engine can only be installed on Mac")
+    s3 = boto3.resource("s3")
+    target_base_dir = "/Applications/Unity/Hub/Editor/{}/PlaybackEngines".format(_unity_version())
```
Comment (AlvaroHG) [inline tasks.py:1125]: Perhaps using the `_unity_path()` function here to not have multiple places to change if this string changes.

**Comment by AlvaroHG** [comment]
One minor comment, LGTM. Awesome to have CloudRendering support!

**Review by mattdeitke — APPROVED** [review]
LGTM, have tested it with the unreleased updates, and all is well :)

**Pull request merged**